stable/odoo/values.yaml

```diff
@@ -66,6 +66,10 @@ postgresql:
   ##
   persistence:
     enabled: true
+    ## If you want to reuse an existing claim, you can pass the name of the PVC using
+    ## the existingClaim variable
```

Comment:
Add an "empty" line (only `##`) between the comment and the value

Reply:
👍